examples/climb_run_qwen2_5_vl_7b_engaging.sh

```diff
@@ -10,4 +11,4 @@ python -m verl.trainer.main \
     algorithm.adv_estimator=grpo \
     data.format_prompt="${format_prompt}" \
     worker.actor.model.model_path=Qwen/Qwen2.5-VL-7B-Instruct \
-    trainer.n_gpus_per_node=4
+    trainer.n_gpus_per_node=1
```

Comment:
I think we load the prompt from .ninja file now, so we no longer need the format_prompt variable here.